### DISCUSSION TITLE
Adding PHP CS Fixer to the project

### DISCUSSION THREAD

**Discussion by JaksyMarci** [JaksyMarci/Cherry-Blossom-Diner#26] [Ideas]
As an extra tool, I would like to add this to our project
cs.symfony.com
It would analyze our php code and make it follow the latest php standards. Any thoughts?

**Comment by JaksyMarci** [comment]
I've created a branch with the tool added, and ran the linter on all the files.

**Comment by StavniczkiBalazs** [comment]
Great tool. Thx Jaksy.
